### DISCUSSION TITLE
Disable LSP formatting on save per-project

### DISCUSSION THREAD

**Discussion by redxtech** [LazyVim/LazyVim#837] [Q&A]
I have a project that I want to disable formatting on save, while keeping it active for everything else.

It's being formatted with `tsserver`, and I've tried many things with per-project `neoconf` files.

I've tried `lspconfig.tsserver.typescript.format.enable = false`, `lspconfig.tsserver.javascript.format.enable = false`, `lspconfig.tsserver.format.enable = false`.

Not sure what else to try.

Is this possible, or am I missing something?

**Comment by folke** [comment]
Add an autocmd that does `require("lazyvim.plugins.lsp.format").autoformat = false` when in your project
